target/scripts/startup/setup.d/security/misc.sh

```diff
@@ -81,12 +81,12 @@ function __setup__security__spamassassin() {
     # shellcheck disable=SC2016
     sed -i -r 's|^\$sa_kill_level_deflt (.*);|\$sa_kill_level_deflt = '"${SA_KILL}"';|g' /etc/amavis/conf.d/20-debian_defaults
 
-    if [[ ${SA_SPAM_SUBJECT} == 'undef' ]]; then
+    if [[ -z ${SPAM_SUBJECT} ]]; then
       # shellcheck disable=SC2016
       sed -i -r 's|^\$sa_spam_subject_tag (.*);|\$sa_spam_subject_tag = undef;|g' /etc/amavis/conf.d/20-debian_defaults
     else
       # shellcheck disable=SC2016
-      sed -i -r 's|^\$sa_spam_subject_tag (.*);|\$sa_spam_subject_tag = '"'${SA_SPAM_SUBJECT}'"';|g' /etc/amavis/conf.d/20-debian_defaults
+      sed -i -r 's|^\$sa_spam_subject_tag (.*);|\$sa_spam_subject_tag = '"'${SPAM_SUBJECT}'"';|g' /etc/amavis/conf.d/20-debian_defaults
```

Comment:
I guess this is why you still have an rspamd specific approach with the sieve script. In which case my changelog suggestion for breaking changes is a bit inaccurate.

It should be fine to ensure the config is undef here, and use the same sieve script though, as the changelog suggests?

The main benefit of keeping this logic is `_SCORE_` tag will still be usable for Amavis+SpamAssassin, and if there was a DMS delivering spam to a different service rather than our bundled Dovecot service, it'd still keep the spam prefix in the subject.

Reply:
I will change to only Sieve as the only mechanism with the next commits, hence your suggestions for the changelog are correct. The `_SCORE_` logic will be removed as well; users that require it can manually patch the configuration.